### ISSUE TITLE
Server do not receive logs from agent

### ISSUE THREAD

**Issue by YjyJeff** [ossec/ossec-hids#1746]
Hi, 
I am deploying ossec-hids on my two computers, one as an agent and another as a server. I want to view the logs that collected from agent, therefore, I set \<logall\> to yes in my server's ossec.conf file. However, I still did not see any logs from the agent. I am sure the agent has connected with server, because I can find the log of rootcheck  in server's /var/ossec/logs/archives/archives.log file.

 Here is part of  my agent's ossec.conf:

 >\<localfile\>
    <log_format>syslog</log_format>
    \<location>/var/log/auth.log\</location>
  \</localfile\> 

  
 
As you can see, I want to monitor the agent's auth.log. But I did not find any log about agent's auth.log in server's /var/ossec/logs/archives/archives.log. What should I do? Thanks in advance!

**Comment by ddpbsd** [comment]
Are there any hints in the agent's ossec.log?
Are all of the necessary OSSEC processes running (`/var/ossec/bin/ossec-control status`) on the agent?

**Comment by YjyJeff** [comment]
> Are there any hints in the agent's ossec.log?
> Are all of the necessary OSSEC processes running (`/var/ossec/bin/ossec-control status`) on the agent?

Yes,  the agent's ossec.log said ubable to send message to server 0.0
![Screen Shot 2019-07-16 at 7 50 09 PM](https://user-images.githubusercontent.com/12508308/61292163-17d95c80-a803-11e9-82ed-128c3517307c.png)

But I can ping the server successfully. And some logs can sent to server.

All of necessary OSSEC processes are running well.
![Screen Shot 2019-07-16 at 7 53 03 PM](https://user-images.githubusercontent.com/12508308/61292284-625ad900-a803-11e9-983d-ca327867fbd9.png)

**Comment by ddpbsd** [comment]
Look for similar errors in the OSSEC server's ossec.log.
You might have to turn on debugging on the server: `/var/ossec/bin/ossec-control enable debug && /var/ossec/bin/ossec-control restart` 
You can use `tcpdump` on the OSSEC server to see if the packets from the agent are making it to the OSSEC server. Make sure they look like they are coming from the same IP address that was configured for the agent with `manage_agents`.
Also look for responses from the OSSEC server to the agent.

**Comment by AATHITH** [comment]
I'm facing the same issue as mentioned by @ddpbsd  now on Dec, 2024!

**Comment by atomicturtle** [comment]
Redirecting questions to the discussions area. That said,  checking with a sniffer is the next stop. Its most likely that an upstream filter is blocking traffic.
